src/DynamoRevit/ViewModel/DynamoRevitViewModel.cs

```diff
@@ -0,0 +1,125 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using Dynamo.Applications.Models;
+using Dynamo.Interfaces;
+using Dynamo.Models;
+using Dynamo.ViewModels;
+using Dynamo.Wpf.Interfaces;
+using Dynamo.Wpf.UI;
+using Dynamo.Wpf.ViewModels.Core;
+
+using RevitServices.Persistence;
+
+namespace Dynamo.Applications.ViewModel
+{
+    public class DynamoRevitViewModel : DynamoViewModel, IDisposable
+    {
+        protected DynamoRevitViewModel(
+            DynamoModel dynamoModel,
+            IWatchHandler watchHandler,
+            IVisualizationManager vizManager,
+            string commandFilePath,
+            IBrandingResourceProvider resourceProvider,
+            bool showLogin = false) :
```

Comment:
Maybe this is an opportunity to give DynamoViewModel's constructor some cleanup (i.e. a single parameter)?